### PR TITLE
feat: Update depth presets for Capri

### PR DESCRIPTION
**Pull request by knipec**
For depth presets:
- remove LandPKS
- add BLM monitoring standard
- make NRCS the default
- update tests

### Related Issues
Client-shared part to address [#1541](https://github.com/techmatters/terraso-mobile-client/issues/1541)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "munsell": "^1.1.5",
     "react": "18.2.0",
     "react-redux": "^8.1.3",
-    "terraso-backend": "github:techmatters/terraso-backend#b542062",
+    "terraso-backend": "github:techmatters/terraso-backend#9618c528cce18a9bface8e706219df1aa4452c78",
```
Comment (knipec) [inline package.json:15]: Note to self: Update this & npm install when backend PR gets merged

File: src/constants.ts [hunk 2]
```diff
@@ -47,10 +39,17 @@ export const DEPTH_INTERVAL_PRESETS = mapEntries(
       { start: 60, end: 100 },
       { start: 100, end: 200 },
     ],
+    BLM_STANDARD: [
```
Comment (paulschreiber) [inline src/constants.ts:42]:
```suggestion
    BLM: [
```

File: src/soilId/soilIdTypes.ts [hunk 3]
```diff
@@ -167,8 +167,8 @@ export const surfaceCracks = [
 export type ProjectDepthIntervalPreset =
   SoilIdProjectSoilSettingsDepthIntervalPresetChoices;
 export const DEPTH_PRESETS = [
-  'LANDPKS',
   'NRCS',
+  'BLM_STANDARD',
```
Comment (paulschreiber) [inline src/soilId/soilIdTypes.ts:171]:
```suggestion
  'BLM',
```

File: src/selectors.ts [hunk 4]
```diff
@@ -237,8 +237,8 @@ const projectIntervals = (settings: ProjectSoilSettings) => {
 
 const sitePresetIntervals = (soilData: SoilData) => {
   switch (soilData.depthIntervalPreset) {
-    case 'LANDPKS':
     case 'NRCS':
+    case 'BLM_STANDARD':
```
Comment (paulschreiber) [inline src/selectors.ts:241]:
```suggestion
    case 'BLM':
```

File: src/selectors.ts [hunk 5]
```diff
@@ -225,8 +225,8 @@ export const selectUserRoleProject = createSelector(
 
 const projectIntervals = (settings: ProjectSoilSettings) => {
   switch (settings.depthIntervalPreset) {
-    case 'LANDPKS':
     case 'NRCS':
+    case 'BLM_STANDARD':
```
Comment (paulschreiber) [inline src/selectors.ts:229]:
```suggestion
    case 'BLM':
```

File: src/constants.ts [hunk 6]
```diff
@@ -47,10 +39,17 @@ export const DEPTH_INTERVAL_PRESETS = mapEntries(
...
+      { start: 1, end: 10 },
+      { start: 10, end: 20 },
+      { start: 20, end: 50 },
+      { start: 50, end: 70 },
+    ],
   } as const,
   depthIntervals =>
     depthIntervals.map(depthInterval => ({ label: '', depthInterval })),
-) satisfies Record<'LANDPKS' | 'NRCS', readonly LabelledDepthInterval[]>;
+) satisfies Record<'NRCS' | 'BLM_STANDARD', readonly LabelledDepthInterval[]>;
```
Comment (paulschreiber) [inline src/constants.ts:52]:
```suggestion
) satisfies Record<'NRCS' | 'BLM', readonly LabelledDepthInterval[]>;
```

**Review by paulschreiber — APPROVED** [review]
Approved. Recommend removing the `_STANDARD` prefix.

**Review by shrouxm — APPROVED** [review]

**Pull request merged**